src/OrbitGl/CaptureWindowTest.cpp

```diff
@@ -32,74 +33,81 @@ constexpr int kViewportHeight = 100;
 constexpr double kSliderPosEpsilon = 0.0001;
 constexpr double kTimeEpsilonUs = 0.0000001;
 
-class NavigationTestCaptureWindow : public CaptureWindow, public testing::Test {
+class NavigationTestCaptureWindow : public testing::Test {
  public:
-  explicit NavigationTestCaptureWindow() : CaptureWindow(nullptr) {
-    capture_client_app_ = &capture_client_app_fake_;
-
-    Resize(kViewportWidth, kViewportHeight);
+  explicit NavigationTestCaptureWindow() : capture_window_{nullptr, &capture_client_app_fake_} {
+    capture_window_.Resize(kViewportWidth, kViewportHeight);
```

Comment:
This is the only change I had to make to `CaptureWindow`. It now takes the `CaptureControlInterface` as a separate constructor argument. This avoids that we need to overwrite the protected variable (previously in line 38).

Reply:
Check out GlCanvas.cpp to see on how this is handled now for the non-test case.